src/plugins/intel_cpu/src/nodes/interaction.cpp

```diff
@@ -94,7 +236,7 @@ static inline void flat_triangle(const uint8_t* in, uint8_t* out, size_t size, s
     }
 }
 
-void Interaction::execRef(dnnl::stream strm) {
+void Interaction::execRef(dnnl::stream strm, bool fuseFQ) {
```

Comment:
why you need new arg? Just check `fqScales.empty()` on line 254

Reply:
I think this could save one comparison.

Reply:
remove that arg since we implement same jit kernel for fp32 & int8